### ISSUE TITLE
[Bug]: compile error on media-driver 24.3.0

### ISSUE THREAD

**Issue by heitbaum** [intel/media-driver#1834]
### Which component impacted?

Build

### Is it regression? Good in old configuration?

Yes, it's good in old version

### What happened?

Building media-driver 24.3.0 fails as shown in the logs

### What's the usage scenario when you are seeing the problem?

Others

### What impacted?

Compile error with gcc-14.1.0

### Debug Information

```
../media_softlet/agnostic/common/shared/packet/media_render_cmd_packet.cpp: In member function 'MOS_STATUS RenderCmdPacket::PrepareComputeWalkerParams(KERNEL_WALKER_PARAMS, MHW_GPGPU_WALKER_PARAMS&)':
../media_softlet/agnostic/common/shared/packet/media_render_cmd_packet.cpp:1267:39: error: storing the address of local variable 'params' in '*gpgpuWalker._MHW_GPGPU_WALKER_PARAMS::inlineDataParamBase' [-Werror=dangling-pointer=]
 1267 |     gpgpuWalker.inlineDataParamBase   = params.inlineDataParams;
      |     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~
../media_softlet/agnostic/common/shared/packet/media_render_cmd_packet.cpp:1199:77: note: 'params' declared here
 1199 | MOS_STATUS RenderCmdPacket::PrepareComputeWalkerParams(KERNEL_WALKER_PARAMS params, MHW_GPGPU_WALKER_PARAMS &gpgpuWalker)
      |                                                        ~~~~~~~~~~~~~~~~~~~~~^~~~~~
../media_softlet/agnostic/common/shared/packet/media_render_cmd_packet.cpp:1199:110: note: 'gpgpuWalker' declared here
 1199 | MOS_STATUS RenderCmdPacket::PrepareComputeWalkerParams(KERNEL_WALKER_PARAMS params, MHW_GPGPU_WALKER_PARAMS &gpgpuWalker)
      |                                                                                     ~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~
cc1plus: all warnings being treated as errors
[358/1566] Building CXX object media_driver/CMakeFile...tic/common/shared/mediacopy/media_blt_copy_next.cpp.o
ninja: build stopped: subcommand failed.
```

### Do you want to contribute a patch to fix the issue?

No.

**Comment by heitbaum** [comment]
Fix seems to be 47bae3c809c9f5b11793b76b5edfd7704adb4080

**Comment by heitbaum** [comment]
Closing - fixed in 24.3.1
